frontend/src/components/tokenHandler.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+export const TokenStateHandler = () => {
+    const [token, setTokenInState] = useState();
+    return {
+        token,
+        setTokenInState
+    }
+}
+
+export const getTokenFromStorage = () => {
+    return sessionStorage.getItem("token");
+}
+
+export const writeToken = (token) => (
+    sessionStorage.setItem("token", token)
+)
+
+export const deleteToken = () => {
+    sessionStorage.removeItem("token")
```

Comment:
Ezeket a hook-on keresztül kellene használni, nem exporttal máshol, mert például törlésnél a state-et is setelni kellene.